docs/user/private-app-repository.md

```diff
@@ -101,6 +101,14 @@ To add Harbor as the private chart repository, go to `Configuration > App Reposi
 
 Once you create the repository you can click on the link for the specific repository and you will be able to deploy your own applications using Kubeapps.
 
+### Harbor: Authentication/Authorization
+
+It is possible to configure Harbor to use authentication:
+
+- Using HTTP [basic authentication](https://github.com/goharbor/harbor/blob/master/docs/user_guide.md#working-with-helm-cli). To use this feature:
+  - When creating a new project for serving as the helm chart repository in Harbor, set the `Access Level` of the project to non public. This enforces authentication to access the charts in the chart repository via Helm CLI or other clients.
+  - When `Adding App Repository` in Kubeapps, set the `Authorization Header` to the `Basic` auth header using the base64 encoded string of `user:password`. For example, for the user "admin" and password "Harbor12345", use the following auth header: `Basic YWRtaW46SGFyYm9yMTIzNDU=`.
```

Comment:
I have just merged a new feature that allows you to just select "Basic Auth" in the form and fill username and password (see other screenshots)

Reply:
@andresmgot, Got it. This new feature is good. I will update the above statement.